content/docs/reference-react.md

```diff
@@ -70,8 +70,7 @@ See the [React.Component API Reference](/docs/react-component.html) for a list o
 * * *
 
 ### `React.PureComponent`
-
-`React.PureComponent` is exactly like [`React.Component`](#reactcomponent), but implements [`shouldComponentUpdate()`](/docs/react-component.html#shouldcomponentupdate) with a shallow prop and state comparison, which [`React.Component`](#reactcomponent) is missing completely. 
+`React.PureComponent` is similar to [`React.Component`](#reactcomponent). The difference between them is that [`React.Component`](#reactcomponent) doesn't implement [`shouldComponentUpdate()`](/docs/react-component.html#shouldcomponentupdate), but `React.PureComponent` implements it with a shallow prop and state comparison. 
```

Comment:
I would leave out the part about React.Component doesn't implement sCU, as it could be confused with saying that it can't/shouldn't. It should be enough to say: 

>`React.PureComponent` implements `shouldComponentUpdate` with a shallow prop and state comparison.

Reply:
It sounds like this was the main reason Szymon opened the PR in the first place. (He had an incorrect impression about `Component` implementing a deep-comparison `shouldComponentUpdate` when it actually doesn't implement one at all.)